### PR TITLE
Significantly modifies file structure for views. Adds view for contac…

### PR DESCRIPTION
**Pull request by FrankieFabuloso**
…t with working for submission

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ var bodyParser = require('body-parser');
 var index = require('./routes/index');
 
 var app = express();
-
+console.log('index:', index)
```
Comment (Moniarchy) [inline app.js]: Why have you got this `console.log()`?

File: public/javascripts/navbar.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+$( document ).ready( function() {
+
+  let pathname = window.location.pathname;
+  alert(pathname)
```
Comment (Moniarchy) [inline public/javascripts/navbar.js]: Also, why this `alert`? Was this just for testing purposes?

File: views/contact/contactForm.pug [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.form-wrapper
+  form.contact-form(action="https://formspree.io/rocha.francisco123@gmail.com" method="POST")
```
Comment (Moniarchy) [inline views/contact/contactForm.pug:2]: i know that you can use that action attribute to specify a module to access; i have never seen this before. Won't this be looking for the POST action for that formspree?

**Review by Moniarchy — COMMENTED** [review]
Thanks for letting me ask questions of your code. :)

File: app.js (hunk 1 above)
Comment (FrankieFabuloso) [reply app.js]: lol I forgot about it.

File: views/contact/contactForm.pug (hunk 3 above)
Comment (FrankieFabuloso) [reply views/contact/contactForm.pug:2]: Im basically using formsprees API. That's how they tell you to use it.

**Pull request merged**